appveyor.yml

```diff
@@ -39,9 +62,10 @@ install:
     - "conda --version"
     - "python --version"
 
-# Not a .NET project, we build SunPy in the install step instead
+# Not a .NET project, we build ci-helpers in the install step instead
 build: false
 
 test_script:
-  - "%CMD_IN_ENV% python --version"
-
+  - "echo %TEST_CMD%"
+  - "%CMD_IN_ENV% python -c %TEST_CMD%"
```

Comment:
I wonder if the issue is that there are quotes missing arond ``%TEST_CMD``? I.e. at the moment it's doing something like

```
python -c import numpy ...
```

instead of

```
python -c 'import numpy ...'
```

?